src/main/java/org/springframework/integration/kafka/core/Partition.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.integration.kafka.core;
+
+/**
+ * A reference to a Kafka partition, with both topic and partition id
+ *
+ * @author Marius Bogoevici
+ */
+public class Partition {
```

Comment:
We could, but I'd rather not have mix items from the Scala API with our own. So I'd rather be consistent on that.